Automattic/wp-calypso
# Add INSTALL_PURCHASED_PLUGINS feature

#### Proposed Changes

* Buying marketplace plugins is now gated behind the new `INSTALL_PURCHASED_PLUGINS` feature, instead of `INSTALL_PLUGINS`.
  * The backend has already been updated to provide `INSTALL_PURCHASED_PLUGINS` to the business, ecommerce, and pro plans.

#### Testing Instructions



* Visit `/plugins/<site>` on a variety of simple sites: Pro, Business, Ecommerce, Starter, Free.
  * The Pro, Business and Ecommerce plans should be able to buy and install plugins.
  * Starter and Free should not.
  * **Behavior should be the same as production.**



Related to D82978-code
Related to https://github.com/Automattic/wp-calypso/issues/58998
